src/Datadog.Trace/AppSec/Waf/NativeBindings/Native.cs

```diff
@@ -3,22 +3,82 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-#if !NETFRAMEWORK
-#endif
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Runtime.InteropServices;
 using System.Text;
+using Datadog.Trace.Logging;
 
 namespace Datadog.Trace.AppSec.Waf.NativeBindings
 {
     internal class Native
     {
-#if !NETFRAMEWORK
-        private const string DllName = "Datadog.Trace.ClrProfiler.Native";
+#if NETFRAMEWORK
+        private const string DllName = "Sqreen.dll";
+#else
+        private const string DllName = "Sqreen";
+#endif
```

Comment:
The conditional logic repeated in the construct is because tuple deconstruction is not support in .NET framework 4.5. I have come up with a way to remove the conditional compilation in the construct, but I can't see anyway of avoiding it here.